test/v1/examples.test.js

```diff
@@ -86,7 +87,7 @@ describe('examples', function() {
     //end::configuration[]
 
     var s = driver.session();
-    s.run( "CREATE (p:Person { name: {name} })", {name: "The One"} )
+    s.run( "CREATE (p:Person {name: {name}})", {"name": "The One"} )
```

Comment:
No need to quote js object keys, unless the key includes breaking characters like space etc.